tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/controller.go

```diff
@@ -128,10 +128,12 @@ func initLogger(ctx context.Context, kubeClientSet kubernetes.Interface) *zap.Su
 		MaxSize: 1024 * 100, // TODO make it configurable via a configmap.
 	}
 	enabled, err := loadObjectStoreConfig(ctx, kubeClientSet, &loggerConfig)
-	if err == nil {
+	if err == nil && enabled {
```

Comment:
nit: although the `err` changes inside the condition, but seem this could be merged into the next condition check line 139.